packages/now-next/src/index.ts

```diff
@@ -807,7 +824,10 @@ export const build = async ({
         }
       })
     );
-    console.timeEnd(allLambdasLabel);
+
+    if (hasLambdas) {
+      console.timeEnd(allLambdasLabel);
```

Comment:
The `console.time` call above should also be wrapped in the if statement if you're not going to end it (otherwise a timer exists).

Reply:
Updated